stripe/src/test/java/com/stripe/android/networking/AnalyticsRequestExecutorTest.kt

```diff
@@ -3,31 +3,51 @@ package com.stripe.android.networking
 import android.content.Context
 import androidx.test.core.app.ApplicationProvider
 import com.google.common.truth.Truth.assertThat
+import com.nhaarman.mockitokotlin2.mock
+import com.nhaarman.mockitokotlin2.verify
 import com.stripe.android.ApiKeyFixtures
+import com.stripe.android.Logger
 import com.stripe.android.model.PaymentMethodCreateParams
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.TestCoroutineDispatcher
 import org.junit.runner.RunWith
 import org.robolectric.RobolectricTestRunner
 import kotlin.test.Test
 
+@ExperimentalCoroutinesApi
 @RunWith(RobolectricTestRunner::class)
 class AnalyticsRequestExecutorTest {
```

Comment:
while you're at it, should we make `connectionFactory` a constructor parameter of `AnalyticsRequestExector.Default`? Then it would be possible to mock it in the test without really touching network

Reply:
I want this to use the real network to verify that we can successfully hit the analytics endpoint

Reply:
IMHO that should be tested in `ConnectionFactoryTest` as it's a replaceable dependency of the Executor and in this test it should just mock good/bad response from network.  But will let you decide as this is test now is not that super complicated:)

Reply:
Yeah that makes sense! Worth auditing which of our tests go across the wire too to make sure we aren't doing too many extraneous network requests in tests.